### PR TITLE
Generate gRPC messages for common proto packages.

### PR DESCRIPTION
**Pull request by jmuk**

### CODE REVIEW

File: templates/commonpb/java/build.gradle.mustache [hunk 1]
```diff
@@ -19,6 +19,7 @@ repositories {
 
 dependencies {
   compile "com.google.protobuf:protobuf-java:3.0.0-beta-3"
+  compile "io.grpc:grpc-all:{{{dependencies.grpc.java.version}}}"
```
Comment (garrettjonesgoogle) [inline templates/commonpb/java/build.gradle.mustache]: protobuf line: {{{dependencies.protobuf.java.version}}}

**Comment by garrettjonesgoogle** [comment]
LGTM, with optional suggestion.

File: templates/commonpb/java/build.gradle.mustache (hunk 1 above)
Comment (jmuk) [inline templates/commonpb/java/build.gradle.mustache]: done

**Comment by bjwatson** [comment]
Thanks @jmuk

**Pull request merged**